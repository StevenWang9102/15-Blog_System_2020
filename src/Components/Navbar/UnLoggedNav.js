import React from "react";
import { HashRouter as Router, Link } from "react-router-dom";

export const UnLoggedNav = props => {
  return (
    <Router>
      <div>
        <li className='nav-item'>
          <Link className='nav-link active' to='/home'>
            Home
          </Link>
        </li>

        <li className='nav-item'>
          <Link className='nav-link' to='/sign_in'>
            Sign in
          </Link>
        </li>

        <li className='nav-item'>
          <Link className='nav-link' to='/sign_up'>
            Sign up
          </Link>
        </li>
      </div>
    </Router>
  );
};
