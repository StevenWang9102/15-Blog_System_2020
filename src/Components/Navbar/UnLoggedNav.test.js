import React from "react";
import { UnLoggedNav } from "./UnLoggedNav";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <UnLoggedNav/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
