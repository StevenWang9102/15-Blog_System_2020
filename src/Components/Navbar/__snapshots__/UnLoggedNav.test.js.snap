// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <li
    className="nav-item"
  >
    <a
      className="nav-link active"
      href="#/home"
      onClick={[Function]}
    >
      Home
    </a>
  </li>
  <li
    className="nav-item"
  >
    <a
      className="nav-link"
      href="#/sign_in"
      onClick={[Function]}
    >
      Sign in
    </a>
  </li>
  <li
    className="nav-item"
  >
    <a
      className="nav-link"
      href="#/sign_up"
      onClick={[Function]}
    >
      Sign up
    </a>
  </li>
</div>
`;
